plugin

```diff
@@ -170,6 +170,7 @@
   "georgezhao2010/lovelace-curtain-card",
   "Gh61/lovelace-hue-like-light-card",
   "giovannilamarmora/lovelace-google-components",
+  "giovannilamarmora/lovelace-material-components",
```

Comment:
Your repository currently only has alpha releases (v2.0.0-ALPHA). HACS default list requires stable releases. Pre-release versions (alpha, beta, RC) are not considered stable releases for HACS inclusion.

**What you need to do:**

1. Complete development and testing to ensure the plugin is production-ready
2. Create a stable release, without pre-release identifiers
3. Once you have published a stable release, mark this PR as ready for review

Reference: https://www.hacs.xyz/docs/publish/include/